### PR TITLE
chore(recommendations-plugin): Use upper tsconfig file

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
recommendations plug-in was using a custom (one that is added by the yeoman generator) while it should use upper configuration like the other plug-ins

### What issues does this PR fix or reference?
N/A

### How to test this PR?
Recommendations plug-in should work as before

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)
- [x] Optional Companion PR for updating the HappyPath tests is approved and ready to be merged

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### Happy Path Channel

HAPPY_PATH_CHANNEL=next

### CODE REVIEW

File: plugins/recommendations-plugin/tsconfig.json [hunk 1]
```diff
@@ -1,33 +1,19 @@
...
-      "skipDefaultLibCheck": true,
-      "types": ["node", "reflect-metadata", "jest"],
-      "experimentalDecorators": true,
-      "emitDecoratorMetadata": true,
-      "lib": [
-        "es6"
-    ],
-    }
-  ,
+        "target": "es5",
```
Comment (tsmaeder) [inline plugins/recommendations-plugin/tsconfig.json:4]: The other plugins are using "es6" or nothing at all (inherited from the base config) for a target.

**Review by tsmaeder — CHANGES_REQUESTED** [review]
Other than the "es5", lgtm.

File: plugins/recommendations-plugin/tsconfig.json (hunk 1 above)
Comment (benoitf) [reply plugins/recommendations-plugin/tsconfig.json:4]: Well I copy it from containers-plugin:
There are many other references https://github.com/eclipse-che/che-theia/search?q=es5
And it'll be replaced anyway by https://github.com/eclipse-che/che-theia/pull/1213 so it's like uneeded requirement

**Review by tsmaeder — APPROVED** [review]
lgtm

**Pull request merged**